app/javascript/components/close_contact/CloseContact.js

```diff
@@ -120,8 +120,10 @@ class CloseContact extends React.Component {
                   className="form-square"
                   value={this.state.notes || ''}
                   placeholder={this.closeContactNotePlaceholder}
+                  maxLength="2000"
```

Comment:
So how will this work for exiting records that have close contact notes that already exceed this limit? We're just including this limit because we had a limit already in the Exposure notes form control, yes?

Reply:
As it just is an input limit, having the value already be set to a string over `2,000` characters will not truncate the string.
![image](https://user-images.githubusercontent.com/17532163/95990791-f302d900-0df9-11eb-8448-b1e11a7f6846.png)


(Note the negative character limit).